packages/myst-frontmatter/src/exports/types.ts

```diff
@@ -22,9 +24,12 @@ export type Export = {
   template?: string | null;
   output?: string;
   zip?: boolean;
-  toc?: string;
+  toc?: TOC;
   articles?: ExportArticle[];
+  top_level?: 'parts' | 'chapters' | 'sections';
```

Comment:
I don't think that this is documented yet? This is only available for LaTeX it seems?

@fwkoch how was this done before that we didn't need to specify this?

Reply:
This was before taken from the toc children names, I.e section vs chapter vs part.

That means the toc was both used for site layout, includes, and exports. Now the toc is only the first two, and we have an export-level flag to govern how sectioning occurs (which makes more sense to me!)